packages/next/package.json

```diff
@@ -216,8 +216,8 @@
     "jest-worker": "27.0.0-next.5",
     "json5": "2.2.1",
     "jsonwebtoken": "8.5.1",
-    "loader-utils2": "npm:loader-utils@2.0.0",
-    "loader-utils3": "npm:loader-utils@3.1.3",
+    "loader-utils2": "npm:loader-utils@^2.0.0",
+    "loader-utils3": "npm:loader-utils@^3.1.3",
```

Comment:
The `^` might make this resolve differently when new versions are released

Reply:
was autoedited by pnpm. I can try to revert it through 👍